Braze-Demo/AppboyManager.swift

```diff
@@ -70,26 +70,23 @@ extension AppboyManager {
   func application(_ application: UIApplication, didReceiveRemoteNotification userInfo: [AnyHashable : Any], fetchCompletionHandler completionHandler: @escaping (UIBackgroundFetchResult) -> Void) {
```

Comment:
I'd recommend naming this file and everything else in the demo `Braze` even if it uses `Appboy`-named APIs. I don't think it's too confusing and you'll save a rename